src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1041,6 +1041,11 @@
       "run": "after",
       "snippetPath": "createInstanceSetView.sql",
       "fromModuleVersion": "25.0.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "removeRelatedInstanceTypeOperations.sql",
+      "fromModuleVersion": "25.0.0"
```

Comment:
Oh, is there another difference that I missed in my write up or is this the `set_md` differences that are the responsibility of RMB?

Reply:
No, I did this before I saw your write-up. Rincy will be removing these.